src/react-apps/applications/service-development/src/features/deploy/components/currentVersionPaper.tsx

```diff
@@ -38,9 +38,14 @@ interface ICurrentVersionPaperProps {
 }
 
 export const CurrentVersionPaper = (props: ICurrentVersionPaperProps) => {
+  const [imageVersion, setImageVersion] = React.useState(props.imageVersion);
   const { classes } = props;
   const { org, service } = window as IAltinnWindow;
 
+  React.useEffect(() => {
+    setImageVersion(props.imageVersion);
+  }, [props.imageVersion]);
```

Comment:
Wouldn't this cause an infinity loop? Since you're saying that you should rerender everytime you use the hook "setImageVersion", and you set the image version in the useEffects? Or is it everytime it changes that this will trigger a re-render? Or have i not understood hooks?

Reply:
It will trigger a rerender when props.imageVersion changes. It should be equivalent  to ComponentWillReceiveProps / ComponentDidUpdate. Just having setImageVersion(props.imageVersion) would probably cause an infinity loop. 

You have to hard refresh the test-page at the moment and this is an attempt to solve it.